### PR TITLE
Sdp coobook fixes

### PR DESCRIPTION
**Pull request by Smithash**
UI fixes and got refresh of cookbook to not crash but its not 100% perfect. when you refresh it goes to first recipe in the cookbook and not the current one that you refreshed on.

### CODE REVIEW

File: src/lib/components/small_recipe.svelte [hunk 1]
```diff
@@ -50,10 +51,9 @@
 	let recipeInstructions = '';
 	$: recipeID && refreshRecipe();
 
-	let recipeExists = false;
-
 	const refreshRecipe = async () => {
 		smallRecipeData = await getSmallRecipeData();
+		console.log(smallRecipeData);
```
Comment (orwellian225) [inline src/lib/components/small_recipe.svelte:56]: Pls remove console.logs

File: src/routes/(app)/cookbook/[cookbook_id]/+page.svelte [hunk 2]
```diff
@@ -81,6 +90,8 @@
 			console.error('An error occurred while deleting the recipe:', error);
 		}
 	};
+
+	console.log(cookbookID);
```
Comment (orwellian225) [inline src/routes/(app)/cookbook/[cookbook_id]/+page.svelte:94]: Pls remove

**Review by orwellian225 — CHANGES_REQUESTED** [review]
Please remove the console.logs but otherwise looks good

**Review by nihal-ranchod — APPROVED** [review]
lgtm

**Review by orwellian225 — APPROVED** [review]
thanks, all good now

**Pull request merged**